components/CreateOrganizationForm.js

```diff
@@ -86,6 +89,9 @@ const CreateOrganizationForm = props => {
     if (values.name.length > 50) {
       errors.name = intl.formatMessage(orgMessages.errorName);
     }
+    if (values.name.length > 50) {
```

Comment:
The variable name is incorrect. I would also challenge the `50` limit, while it's important to have it for the name (because it's displayed everywhere in the interface) the most important thing here is to store the real legal name.

According to https://companieshouse.blog.gov.uk/2019/02/14/symbols-and-characters-in-a-company-name/, it's 160 in the UK. I would keep it safe and limit it to `256` characters.

```suggestion
    if (values.legalName.length > 256) {
```